index.html

```diff
@@ -15,14 +15,14 @@ <h1 id="title">Survery Form</h1>
   <option>Student</option>
   <option>Teacher</option>
  </select><br>
- <!-- added form section-->
+ <!-- added form section
 <input type ="radio"name="Gender" value="Male"/><Label>Male</Label><br>
 <input type = "radio"name ="Gender" value="Female"><Label>Female</Label><br>
 <input type = "checkbox"value="Primary"/><Label>Primary</Label>
 <input type = "checkbox"value="secondary"/>
 <Label>Secondary</Label><br>
 <textarea placeholder="additional comments"></textarea><br>
 <input  id="submit"type="submit"/><br>
-  </Form>
+  </Form>-->
```

Comment:
kindly could you please remove the commenting out form section as its not a good practice to comment out the code